src/main/scala/cognite/spark/v1/SequencesRelation.scala

```diff
@@ -48,9 +48,27 @@ class SequencesRelation(config: RelationConfig)(val sqlContext: SQLContext)
         s.dataSetId
       )
     }
-    client.sequences
-      .create(sequences)
-      .flatTap(_ => incMetrics(itemsCreated, sequences.size)) *> IO.unit
+    val nbColumns = sequences.flatMap(_.columns.toList).size
+    if (nbColumns < Constants.DefaultSequencesTotalColumnsLimit) {
+      client.sequences
+        .create(sequences)
+        .flatTap(_ => incMetrics(itemsCreated, sequences.size)) *> IO.unit
```

Comment:
No need to special case this with the suggestion below, I think.

Reply:
yes 😁